pkg/v22/resource/clusterconfigmap/current.go

```diff
@@ -45,19 +47,25 @@ func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*
 		return nil, nil
 	}
 
-	name := key.ClusterConfigMapName(clusterConfig)
+	var configMaps []*corev1.ConfigMap
+	{
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding configMaps in namespace %#q", key.ClusterID(clusterConfig)))
 
-	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding cluster configMap %#q in namespace %#q", name, clusterConfig.ID))
+		o := metav1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s", label.ManagedBy, project.Name()),
```

Comment:
We now get all configmaps with the label. This excludes user configmaps created via our API or with kubectl.